src/GitVersionTfsTask/GitVersion.ps1

```diff
@@ -54,6 +54,13 @@ if($additionalArguments)
   $argsGitVersion = $argsGitVersion + " $additionalArguments"
 }
 
+if(!$additionalArguments -or !$additionalArguments.Contains("-debug"))
+{
+  if([string]::IsNullOrEmpty($env:SYSTEM_DEBUG) -or $env:SYSTEM_DEBUG -ne "true"){
```

Comment:
This is too generic for a variable name. We should name it `GITVERSION_LOG_LEVEL`